src/database/controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Pool } from 'pg';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+const pool = new Pool({
+  connectionString: process.env.DATABASE_URL,
+});
+
+export const connect = async () => {
```

Comment:
```suggestion
function connect():Promise<identify what should be the return type>{
```

First you've to create a class `DatabaseController`  and then make this change.